job-descriptions/software-engineer.md

```diff
@@ -40,15 +40,20 @@ Take a look at our public [near-term product roadmap](https://docs.sourcegraph.c
 
 ### Interview process
 
-1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU)!
-1.  We schedule a 30 minute call to chat with you about Sourcegraph and to find out what you are looking for in your next role.
+1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU).
+1.  We set up a 30 minute call to chat with you about Sourcegraph to find out what you are looking for in your next role.
 1.  We give you a take home coding assignment which won't take you more than 2 hours to finish.
 1.  We schedule a few more hours of technical and non-technical interviews. We're happy to fly you out to our San Francisco office, or conduct the remaining interviews over video chat, whatever works best for you.
 1.  We check your references.
-1.  We make you a job offer!
+1.  We make you a job offer.
 
-We also expect you to be interviewing us too, so ask us any questions you have along the way!
+We also expect you to be interviewing us too, so ask us any questions you have along the way.
 
 If you aren't ready to start interviewing, but are interested to stop by our San Francisco office to meet the team, let us know!
 
-**[Click here to apply!](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU)**
+
+
+Sourcegraph is an equal opportunity workplace, we embrace diversity and welcome people from all backgrounds and communities.

```

Comment:
I like this wording a lot!

If we want to avoid a [comma splice](https://en.wikipedia.org/wiki/Comma_splice) then the comma needs to be a period, semicolon, or colon.

Reply:
done!